packages/vaex-core/vaex/__init__.py

```diff
@@ -392,6 +392,10 @@ def add(name, column):
         # the first test is to support (partially) pandas 0.23
         if hasattr(pd.core.arrays, 'integer') and isinstance(values, pd.core.arrays.integer.IntegerArray):
             values = np.ma.array(values._data, mask=values._mask)
+        elif hasattr(pd.core.arrays, 'StringArray') and isinstance(values, pd.core.arrays.StringArray):
+            values = np.ma.array(values.to_numpy().astype(np.str), mask=values.isna())
```

Comment:
I think we should go straight to arrow, my guess is the following should work
```suggestion
            values = pa.array(values)
```